src/vivarium_testing_utils/automated_validation/data_transformation/calculations.py

```diff
@@ -57,21 +57,28 @@ def ratio(data: pd.DataFrame, numerator: str, denominator: str) -> pd.DataFrame:
     return series_to_dataframe(data[numerator] / data[denominator])
 
 
-def aggregate_sum(data: pd.DataFrame, groupby_cols: list[str]) -> pd.DataFrame:
+def aggregate(data: pd.DataFrame, groupby_cols: list[str], agg: str) -> pd.DataFrame:
```

Comment:
I feel like sum should be the default for `agg` but I don't feel that strongly about it.